### PR TITLE
[#9430] Added success message for all tablets and single tablet compaction

### PR DESCRIPTION
**Pull request by leeleeian**
There was no notification for completion of all tablets compaction previously. 
This PR adds 2 print statements following the completion of compaction for single tablet and all tablets.

### CODE REVIEW

**Comment by leeleeian** [comment]
@bmatican 
Hi, can you please review this PR? Thanks.

**Review by hulien22 — CHANGES_REQUESTED** [review]
Thanks for your PR @leeleeian ! Left you a few comments on things to fix before we merge this.

**Comment by hulien22** [comment]
Also you can add you github username to [CONTRIBUTORS.md](https://github.com/yugabyte/yugabyte-db/blob/master/CONTRIBUTORS.md) as well :)

**Comment by hulien22** [comment]
Thinking a bit more about this, another thing you could do here would be to add something similar to https://github.com/yugabyte/yugabyte-db/blob/master/src/yb/tools/yb-admin_client.cc#L1435-L1438 at the end of `TsAdminClient::FlushTablets`.

(ie check is_compaction to print "Successfully compacted " or "Successfully flushed ", and then check tablet_id to either print "tablet: <tablet_id>." or just print "all tablets.").

File: src/yb/tools/ts-cli.cc [hunk 1]
```diff
@@ -658,11 +658,13 @@ static int TsCliMain(int argc, char** argv) {
     string tablet_id = argv[2];
     RETURN_NOT_OK_PREPEND_FROM_MAIN(client.FlushTablets(tablet_id, true /* is_compaction */),
                                     "Unable to compact tablet");
+    LOG(INFO) << "tablet " << tablet_id << " successfully compacted";
   } else if (op == kCompactAllTabletsOp) {
     CHECK_ARGC_OR_RETURN_WITH_USAGE(op, 2);
 
     RETURN_NOT_OK_PREPEND_FROM_MAIN(client.FlushTablets(std::string(), true /* is_compaction */),
                                     "Unable to compact all tablets");
+    LOG(INFO) << "All tablets successfully compacted";
```
[the remark this answers is not in the document]
Comment (leeleeian) [reply src/yb/tools/ts-cli.cc]: Hi @hulien22, I've already addressed all the original comments. Regarding the last comment, I will respond later. Thanks.

File: src/yb/tools/ts-cli.cc [hunk 2]
```diff
@@ -658,11 +658,13 @@ static int TsCliMain(int argc, char** argv) {
     string tablet_id = argv[2];
     RETURN_NOT_OK_PREPEND_FROM_MAIN(client.FlushTablets(tablet_id, true /* is_compaction */),
                                     "Unable to compact tablet");
+    std::cout << "Tablet " << tablet_id << " successfully compacted";
```
Comment (hulien22) [inline src/yb/tools/ts-cli.cc:661]: Don't forget to also add `<< std::endl;` at the end for these

**Review by hulien22 — CHANGES_REQUESTED** [review]
Any thoughts on my message above for using `TsAdminClient::FlushTablets`? I think its a bit cleaner and would work for the flush commands in addition :)

File: src/yb/tools/ts-cli.cc (hunk 2 above)
Comment (leeleeian) [reply src/yb/tools/ts-cli.cc]: Hi @hulien22 ,
I've added all your suggested changes, can you review it again? thanks.

File: src/yb/tools/ts-cli.cc [hunk 3]
```diff
@@ -469,6 +469,9 @@ Status TsAdminClient::FlushTablets(const std::string& tablet_id, bool is_compact
     return STATUS(IOError, "Failed to flush tablet: ",
                            resp.error().ShortDebugString());
   }
+  std::cout << "Successfully " << (is_compacted ? "compacted " : "flushed ")
```
Comment (hulien22) [inline src/yb/tools/ts-cli.cc:472]:
```suggestion
  std::cout << "Successfully " << (is_compaction ? "compacted " : "flushed ")
```

File: src/yb/tools/ts-cli.cc [hunk 4]
```diff
@@ -469,6 +469,9 @@ Status TsAdminClient::FlushTablets(const std::string& tablet_id, bool is_compact
...
+            << (tablet_id.empty() ? "all tablets" : "tablet <" << tablet_id << ">")
```
Comment (hulien22) [inline src/yb/tools/ts-cli.cc:473]:
```suggestion
            << (tablet_id.empty() ? "all tablets" : "tablet <" + tablet_id + ">")
```

**Review by hulien22 — COMMENTED** [review]
Looks good, just put in some suggestions to make sure that it compiles properly

File: src/yb/tools/ts-cli.cc (hunk 4 above)
Comment (leeleeian) [reply src/yb/tools/ts-cli.cc]: @hulien22 just made those edits, thank you :)

**Review by hulien22 — APPROVED** [review]
Looks good, passed through our internal tests just to double check nothing else is affected, and those were good. 
Will go ahead and merge this, thanks @leeleeian for your contribution!

**Pull request merged**